qa/tasks/cephfs/test_dump_export_states.py

```diff
@@ -0,0 +1,95 @@
+from teuthology.contextutil import safe_while, MaxWhileTries
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+import logging
+
+log = logging.getLogger(__name__)
+
+class TestDumpExportStates(CephFSTestCase):
+    MDSS_REQUIRED = 2
+    CLIENTS_REQUIRED = 1
+
+    EXPORT_STATES = ['locking', 'discovering', 'freezing', 'prepping', 'warning', 'exporting']
+
+    def setUp(self):
+        CephFSTestCase.setUp(self)
+
+        self.fs.set_max_mds(self.MDSS_REQUIRED)
+        self.status = self.fs.wait_for_daemons()
+
+        self.mount_a.run_shell_payload('mkdir -p test/export')
+
```

Comment:
You can also add a tearDown() method here(after `setUp()`) calling `super().tearDown()`

Reply:
> You can also add a tearDown() method here(after `setUp()`) calling `super().tearDown()`

OK, I have added it.